lib/app.js

```diff
@@ -87,7 +87,8 @@ class Mali extends Emitter {
       if (typeof path === 'object' && path.root && path.file) {
         protoFilePath = path.file
         if (!loadOptions.includeDirs) {
-          loadOptions.includeDirs = [path.root]
+          // Support either multiple or single paths.
+          loadOptions.includeDirs = path.root instanceof Array ? path.root : [path.root]
```

Comment:
It's recommended to use `Array.isArray` instead of `var instanceof Array`. You can dive into the justification from https://stackoverflow.com/questions/22289727/difference-between-using-array-isarray-and-instanceof-array. Can you change this line?

Reply:
For sure make sense thanks already changed.